docs/page/configuration.md

```diff
@@ -105,6 +106,7 @@ settings.
             :cwd nil
             :env '(("DEBUG" . "1"))
             :target-module (expand-file-name "~/src/myapp/.env/bin/myapp")
+            :justMyCode :json-false
```

Comment:
```suggestion
:justMyCode nil
```
`:justMyCode` should be optional and off by default. Also, use, and make sure the user can use, `nil` in such cases since that is more idiomatic in ELisp, which lacks a dedicated false type. json-encoder-specifics shouldn't leak trough.